frontend/src/router/router.jsx

```diff
@@ -30,6 +31,7 @@ const RouterComponent = () => { // eslint-disable-line arrow-body-style
         <PrivateRoute path="/login" loggedOut component={Login} />
         <PrivateRoute path="/register" loggedOut component={Register} />
         <PrivateRoute path="/logout" component={Logout} />
+        <PrivateRoute path="/municipalities" loggedOut component={MetadataByMunicipality} />
```

Comment:
Did you mean to make it so that only not logged in users could view the route? Cus logged in users cant view it now

Reply:
Good point, will fix.